src/components/send/SendContainerV2.js

```diff
@@ -0,0 +1,209 @@
+    clearLocalAlert,
+    accountId
+}) => {
+
+    const [amount, setAmount] = useState('');
+    const [estimatedTotalFees, setEstimatedTotalFees] = useState('0');
+    const [estimatedTotalInNear, setEstimatedTotalInNear] = useState('0');
+    const [parsedAmount, setParsedAmount] = useState('');
+    const [receiverId, setReceiverId] = useState('');
+    const [view, setView] = useState('enterAmount');
+    const [maxAmount, setMaxAmount] = useState(null);
+    const [sendingToken, setSendingToken] = useState(false);
+    const [selectedToken, setSelectedToken] = useState(fungibleTokens[0]);
+
+    useEffect(() => {
+        selectedToken.symbol === 'NEAR' && setSelectedToken(fungibleTokens[0]);
+    }, [availableNearToSend]);
+
+    const handleChangeAmount = (userInputAmount) => {
+        // FIX: Add block when entering more than max decimals allowed
+        setAmount(userInputAmount);
+        setParsedAmount(FTMethods.getParsedTokenAmount(userInputAmount, selectedToken.symbol, selectedToken.decimals));
+        setMaxAmount(false);
+    };
+
+    const handleSetMaxAmount = () => {
+        const formattedTokenAmount = FTMethods.getFormattedTokenAmount(selectedToken.balance, selectedToken.symbol, selectedToken.decimals);
+
+        if (!new BN(formattedTokenAmount).isZero()) {
+            setMaxAmount(true);
+            setAmount(formattedTokenAmount.replace(/,/g, ''));
+            setParsedAmount(selectedToken.balance);
+        }
+    };
+
+    const isValidAmount = () => {
+        const parsedTokenAmount = FTMethods.getParsedTokenAmount(amount, selectedToken.symbol, selectedToken.decimals);
+
+        if (maxAmount) {
+            return true;
+        }
+
+        return !new BN(parsedTokenAmount).isZero() && new BN(parsedTokenAmount).lte(new BN(selectedToken.balance)) && isDecimalString(amount);
+        // TODO: Handle rounding issue that can occur entering exact available amount
+    };
+
+    const handleContinueToEnterReceiver = () => {
+        //TODO: Add amount to URL?
+        setView('enterReceiver');
+    };
+
+    const handleSelectToken = (token) => {
+        //TODO: Add token to URL?
+        setSelectedToken(token);
+        setView('enterAmount');
+        setAmount('');
+        setParsedAmount('');
+        setMaxAmount(false);
+    };
+
+    const enterAmountIsComplete = () => {     
+        return amount && !new BN(selectedToken.balance).isZero() && isValidAmount();
+    };
+
+    const handleContinueToReview = async () => {
+        setView('review');
+
+        if (selectedToken.symbol === 'NEAR') {
+            setEstimatedTotalFees(await FTMethods.getEstimatedTotalFees());
+            setEstimatedTotalInNear(await FTMethods.getEstimatedTotalNearAmount(parsedAmount));
+        } else {
+            // FIX: Currently returns 'Cannot read property 'viewFunction' of undefined'
+            //setEstimatedTotalFees(await FTMethods.getEstimatedTotalFees(selectedToken.contractName, receiverId));
```

Comment:
@marcinbodnar do you have any idea why this view function call from FungibleTokens.js fails?

Reply:
@Patrick1904 `FungibleTokens` methods are part of the wallet class:
https://github.com/near/near-wallet/blob/master/src/utils/wallet.js#L134
so if you want to use methods you use `wallet.FungibleTokens.xxx`.

Also, I'm not sure if we want to have these methods that you added to FungibleTokens class - Daryl was planning to have it like low-level functions. cc @MaximusHaximus 

There are also reducers and actions for these functionalities:
https://github.com/near/near-wallet/pull/1903
https://github.com/near/near-wallet/pull/1914
We can extend it with other actions if needed. 
I'm just not sure if we want to store partial data in the redux state, like estimated fees - if so these methods should be called via redux actions. cc @MaximusHaximus

Reply:
> @Patrick1904 FungibleTokens methods are part of the wallet class:
> https://github.com/near/near-wallet/blob/master/src/utils/wallet.js#L134
> so if you want to use methods you use wallet.FungibleTokens.xxx.

Awesome thanks I missed that.

> Also, I'm not sure if we want to have these methods that you added to FungibleTokens class - Daryl was planning to have it like low-level functions. cc @MaximusHaximus

Yep we had a discussion about this on Friday and decided it would be where fungible token-related methods would live.

> There are also reducers and actions for these functionalities:
> #1903
> #1914
> We can extend it with other actions if needed.
> I'm just not sure if we want to store partial data in the redux state, like estimated fees - if so these methods should be called via redux actions. cc @MaximusHaximus

Right, I think we can skip Redux all together, because estimated fees just need to be calculated every time the user gets to the 'review' view. So no need to put it in redux. Same with checking storage deposit, sending tokens, etc. They're not mutating redux state, so we might as well call them directly without redux. I just have to add error handling in the 'review' component since it's not being handled by redux.